app/models/openid_connect/id_token.rb

```diff
@@ -0,0 +1,34 @@
+class OpenidConnect::IdToken < ActiveRecord::Base
+  belongs_to :authorization
+
+  before_validation :setup, on: :create
+
+  default_scope -> { where("expires_at >= ?", Time.now.utc) }
+
+  def setup
+    self.expires_at = 30.minutes.from_now
+  end
+
+  def to_jwt(options={})
+    to_response_object(options).to_jwt OpenidConnect::IdTokenConfig.private_key
+  end
+
+  def to_response_object(options={})
```

Comment:
It's saying the method is too complex and would be better split into more smaller ones.